corehq/apps/userreports/templates/userreports/reportbuilder/configure_report.html

```diff
@@ -47,7 +47,7 @@
                 window.analytics.usage.apply(this, {{event|JSON}});
             {% endfor %}
             window.analytics.trackUsageLink(
-                "#delete-report-button", "Report Builder", "Delete a Repot", reportType
+                "#delete-report-button", "Report Builder", "Delete a Report", reportType
```

Comment:
We might want to leave this typo to maintain consistency in the analytics system. Not sure if it is easy to combine two events or not...